src/applications/vaos/components/cancel/CancelCernerAppointmentModal.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Modal from '@department-of-veterans-affairs/formation-react/Modal';
+import { FETCH_STATUS } from '../../utils/constants';
+import LoadingButton from 'platform/site-wide/loading-button/LoadingButton';
+
+export default function CancelCernerAppointmentModal({
+  onClose,
+  onConfirm,
+  status,
+}) {
+  return (
+    <Modal
+      id="cancelCernerAppt"
+      status="warning"
+      visible
+      onClose={onClose}
+      title="You cannot cancel this appointment here."
+    >
+      To cancel this appointment, sign in on My VA health.
+      <p className="vads-u-margin-top--2">
+        <LoadingButton
+          isLoading={status === FETCH_STATUS.loading}
+          onClick={onConfirm}
+        >
+          Yes, go to My VA Health
+        </LoadingButton>
+        <button
+          className="usa-button-secondary"
+          onClick={onClose}
+          disabled={status === FETCH_STATUS.loading}
+        >
+          No, take me back
+        </button>
```

Comment:
This looks good for the most part.  Just a couple suggestions:

I might be missing something, but I don't think we need to use a `<LoadingButton>` or check `status` to disable the close button here.  I think the `status` was meant for waiting for the async API call for cancel to complete.  Opening Cerner in a new tab is synchronous so we won't have to wait any status to change.

Also, you might not need to have a new action for `closeCancelCernerAppointment()` since it really isn't doing too much.  You could maybe do something like the following, passing the existing `closeCancelAppointment()` action as a prop:

```jsx
<button
  className="usa-primary"
  onClick={() => 
    window.open(getCernerPortalLink());
    onClose();
  }
>
  Yes, go to My VA Health
</button>
```

Reply:
Done